pootle/apps/pootle_project/models.py

```diff
@@ -339,13 +337,16 @@ def resources(self):
         '''
         cursor = connection.cursor()
         cursor.execute(sql_query, [resources_path, resources_path])
-
         results = cursor.fetchall()
 
         # Flatten tuple and sort in a list
-        resources = list(reduce(lambda x,y: x+y, results))
-        resources.sort(key=get_path_sortkey)
+        resources = set()
+        for result in results:
+            s = result[0]
```

Comment:
Check this out https://github.com/julen/pootle/commit/32990d94e52d1101b4c6b656740a971ec9f4ffc0